include/zephyr/bluetooth/mesh/main.h

```diff
@@ -332,6 +348,19 @@ int bt_mesh_input_string(const char *str);
  *  @return Zero on success or (negative) error code otherwise.
  */
 int bt_mesh_input_number(uint32_t num);
+#endif
+
+/** @brief Provide provisioning input OOB numeric value.
+ *
+ *  This is intended to be called after the bt_mesh_prov input callback
```

Comment:
```suggestion
 *  This is intended to be called after the @ref bt_mesh_prov.input callback
```
Same can be done for `bt_mesh_input_number`.